flac.go

```diff
@@ -343,7 +343,7 @@ func (stream *Stream) Seek(sampleNum uint64) (uint64, error) {
 		if err != nil {
```

Comment:
I don't have time to update my PR right now but the `stream.ParseNext()` can just be a `stream.Next()` if I'm not mistaken.